pkg/resources/kafka/pod.go

```diff
@@ -34,7 +34,7 @@ import (
 	pkicommon "github.com/banzaicloud/koperator/pkg/util/pki"
 )
 
-func (r *Reconciler) pod(id int32, brokerConfig *v1beta1.BrokerConfig, pvcs []corev1.PersistentVolumeClaim, log logr.Logger) runtime.Object {
+func (r *Reconciler) pod(id int32, brokerConfig *v1beta1.BrokerConfig, listenerConfig *v1beta1.ListenersConfig, pvcs []corev1.PersistentVolumeClaim, log logr.Logger) runtime.Object {
```

Comment:
Why do we need to add listenerConfig through parameter, I think that information is available through r.KafkaCLuster.ListenerConfig

Reply:
True, nice catch!